docs/started/daemons.md

```diff
@@ -0,0 +1,92 @@
+rucio-cache-consumer | Replica | Adds and deletes cache replicas to the Rucio catalog | [Details](https://rucio.cern.ch/documentation/bin/rucio-cache-consumer)
+rucio-conveyor-finisher | Transfer | Updates Rucio internal state after the file transfer has finished | [Details](https://rucio.cern.ch/documentation/bin/rucio-conveyor-finisher)
+rucio-conveyor-poller | Transfer | Polls updates from the transfer tool to check the transfer state | [Details](https://rucio.cern.ch/documentation/bin/rucio-conveyor-poller)
+rucio-conveyor-preparer | Transer | Prepares data transfers | [Details](https://rucio.cern.ch/documentation/bin/rucio-conveyor-preparer)
+rucio-conveyor-receiver | Transfer | Sister of polller, instead of polling for updates, it reads transfer tools notifications to check transfer state | [Details](https://rucio.cern.ch/documentation/bin/rucio-conveyor-receiver)
+rucio-conveyor-stager | Transfer| Issues staging (bring online) requests to tape RSEs  | [Details](https://rucio.cern.ch/documentation/bin/rucio-conveyor-stager)
+rucio-conveyor-submitter | Transfer| Submit transfer requests to the transfer tool (prepares the transfer as well if the converyor-preparer is not enabled) | [Details](https://rucio.cern.ch/documentation/bin/rucio-conveyor-submitter)
+rucio-conveyor-throttler | Transfer | Queues transfer requests inside Rucio, applying limits, ex: ony one transfer at a time, etc ... | [Details](https://rucio.cern.ch/documentation/bin/rucio-conveyor-throttler)
+rucio-dark-reaper | Deletion | Deletes quarantained replicas | [Details](https://rucio.cern.ch/documentation/bin/rucio-dark-reaper)
+rucio-dumper | Consistency | Dumps file lists. The rucio-auditor consumes these dumps to discover dark data | [Details](https://rucio.cern.ch/documentation/bin/rucio-dumper)
+rucio-follower | Telemetry | Aggregates events affecting DIDs | [Details](https://rucio.cern.ch/documentation/bin/rucio-follower)
+rucio-hermes | Telemtry | Sends Rucio messages to external services (influxDB, OpenSearch, ActiveMQ, ...) | [Details](https://rucio.cern.ch/documentation/bin/rucio-hermes)
+rucio-judge-cleaner | Rule | Cleans expired replication rules | [Details](https://rucio.cern.ch/documentation/bin/rucio-judge-cleaner)
+rucio-judge-evaluator | Rule | Creates and evaluates replication rules based on their state (OK/REPL/STUCK)  | [Details](https://rucio.cern.ch/documentation/bin/rucio-judge-evaluator)
+rucio-judge-injector | Rule | Asynchronously injects replication rules  | [Details](https://rucio.cern.ch/documentation/bin/rucio-judge-injector)
+rucio-judge-repairer | Rule | Repairs stuck replication rules (STATE=STUCK) | [Details](https://rucio.cern.ch/documentation/bin/rucio-judge-repairer)
+rucio-kronos | Telemetry | Consumes Rucio tracing messages and updates access time of replicas | [Details](https://rucio.cern.ch/documentation/bin/rucio-kronos)
```

Comment:
Also updates the access count on the DID, not sure if we want to mention that though